src/diffusers/modeling_utils.py

```diff
@@ -293,33 +294,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         from_auto_class = kwargs.pop("_from_auto", False)
         torch_dtype = kwargs.pop("torch_dtype", None)
         subfolder = kwargs.pop("subfolder", None)
+        device_map = kwargs.pop("device_map", None)
```

Comment:
Sorry I oversaw this the first time. 
@piEsposito could you also add some docstring here?

Reply:
E.g. 3,4 lines under line 264